psydac/api/fem.py

```diff
@@ -1333,7 +1333,7 @@ def construct_arguments(self):
         tests_degrees = flatten(tests_degrees)
         spans         = flatten(spans)
         quads         = flatten(list(zip(points, weights)))
-        quads_degree  = flatten(self.grid.quad_order)
+        quads_degree  = flatten(self.grid.nquads)
```

Comment:
Should we not call this variable `nquads`?

Reply:
Yes, you're right!

Reply:
4cd8ba09c1f3366bd21a0453990f79f50ac9889f

Reply:
There are a total of four more instances of `quads_degree` in the code that I find difficult to understand. While "degree" might be misleading again, to me it seemed like these variables are different from the `nquads` variables so far.

Reply:
Can you point to the code here?

Reply:
@jowezarek 👋

Reply:
> Can you point to the code here?

See new comment attached to the file api/ast/fem.py.